code/datums/diseases/advance/symptoms/fever.dm

```diff
@@ -61,7 +61,10 @@ Bonus
  * * datum/disease/advance/A The disease applying the symptom
  */
 /datum/symptom/fever/proc/set_body_temp(mob/living/M, datum/disease/advance/A)
-	M.add_body_temperature_change("fever", (6 * power) * A.stage)
+	if(!unsafe)
+		M.add_body_temperature_change("fever", max((6 * power) * A.stage, (BODYTEMP_HEAT_DAMAGE_LIMIT - 1)))
+	else
+		M.add_body_temperature_change("fever", max((6 * power) * A.stage, (BODYTEMP_HEAT_DAMAGE_LIMIT + 20)))
```

Comment:
So unsafe fevers change your temp faster

Reply:
fever and chill's raise or lower your body temperature normal causing your natural stabilization what causes you to have issues.
An unsafe chills/fever allows the temperature range to go in to the damage ranges of temperature change.
This is a fix for the massive temperature changes that existed before with fever/chills where we had no cap and the unsafe setting was not used.